frontend/src/components/BattleLogItem.tsx

```diff
@@ -22,6 +24,7 @@ const BattleLogItem: React.FunctionComponent<Props> = ({
 
   return (
     <ListGroupItem
+      className={visible ? "battle-log-item--hidden" : ""}
```

Comment:
I think comparison above is wrong, and this should be the other way round? If visible then hidden?